tests/testthat/test-bm-tpc-h.R

```diff
@@ -140,3 +140,29 @@ ORDER BY
   # clean up
   DBI::dbDisconnect(con_one, shutdown = TRUE)
 })
+
+
+test_that("grepl queries work with dbplyr style sql", {
+  skip_if_not_installed("duckdb", minimum_version = "0.3.3")
+  run <- run_benchmark(
+    tpc_h,
+    engine = "duckdb",
+    query_id = c(2, 9, 13, 14, 16, 20),
+    scale_factor = 1
+  )
+  no_fails <- any(vapply(run, function(x) is.null(x$error), logical(1)))
+  expect_true(no_fails)
+})
+
+
+test_that("query 21 use of logicals passed",{
```

Comment:
```suggestion
test_that("query 21 use of logicals passed", {
```

Very minor style nit